packages/components/src/components/data-list/mobile-description.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Text from '../text';
+
+const MobileDescription = ({ desc, desc_titles }) => {
+    if (desc.message.substring(0, 7) === 'Address') {
+        const address = desc.message.split(', ')[0].split(': ')[1];
+        const transaction = desc.message.split(', ')[1].split(': ')[1];
```

Comment:
same here. if any message coming from API needs to be changed, they should change it on API side as we can't handle it for non-english